src/absil/il.fs

```diff
@@ -1551,41 +1584,51 @@ type ILTypeDefKind =
     | Enum 
     | Delegate
 
+let typeKindOfFlags nm _mdefs _fdefs (super:ILType option) flags =
+    if (flags &&& 0x00000020) <> 0x0 then ILTypeDefKind.Interface 
+    else 
+         let isEnum = (match super with None -> false | Some ty -> ty.TypeSpec.Name = "System.Enum")
```

Comment:
here super is extracted multiple times. Can you please rewrite it so that `match super` is done only once?

Reply:
thanks I just copy that from somewhere else I'll fix that